GeoLib/IO/XmlIO/Qt/XmlStnInterface.cpp

```diff
@@ -250,8 +249,8 @@ bool XmlStnInterface::write()
     root.setAttribute( "xmlns:xsi", "http://www.w3.org/2001/XMLSchema-instance" );
 
     const std::vector<GeoLib::Point*>* stations (_geo_objs.getStationVec(_exportName));
-    bool isBorehole = static_cast<GeoLib::Station*>((*stations)[0])->type() ==
-                      GeoLib::Station::StationType::BOREHOLE;
+    bool is_borehole = static_cast<GeoLib::Station*>((*stations)[0])->type() ==
```

Comment:
not sure, if modified elsewhere.
```suggestion
    bool const is_borehole = static_cast<GeoLib::Station*>((*stations)[0])->type() ==
```